helm/charts/hpe-csi-driver/README.md

```diff
@@ -6,10 +6,9 @@ The [HPE CSI Driver for Kubernetes](https://github.com/hpe-storage/csi-driver) l
 
 - Upstream Kubernetes version > 1.13
 - Other Kubernetes distributions supported
-- OpenShift 4.2 in Tech Preview (3.x is not supported, see [FlexVolume Driver Helm chart](https://github.com/hpe-storage/co-deployments/tree/master/helm/charts/hpe-flexvolume-driver))
-- More distributions will be listed as tests are ongoing
+- OpenShift 4.2, 4.3(RHCOS, RHEL)
 - Recent Ubuntu, CentOS or RHEL compute nodes connected to their respective official package repositories
-- Helm 3.x is not supported with v1.0.0 release.
+- Helm (v3 is supported only from HPE CSI Driver version 1.1.0 onwards)
```

Comment:
Can't we enumerate it like this?
- Helm 2 (whatever caveat if there is one)
- Helm 3 (whatever caveat if there is one)